mantle/kola/tests/ignition/qemufailure.go

```diff
@@ -101,3 +119,104 @@ func ignitionFailure(c cluster.TestCluster) error {
 		return nil
 	}
 }
+
+// Attach a block device which has a pre-existing boot filesystem on it
+func bootfsFailure(c cluster.TestCluster) error {
+	builder := platform.NewQemuBuilder()
+	// inserting log file for troubleshooting
+	// file located in coreos-assembler directory
+	builder.InheritConsole = console
+	if !console {
+		builder.ConsoleFile = "console.txt"
+		fmt.Println("Console log saved...")
+	}
+
+	failConfig, err := conf.EmptyIgnition().Render(conf.FailWarnings)
+	if err != nil {
+		return errors.Wrapf(err, "creating empty config")
+	}
+	defer builder.Close()
+	builder.SetConfig(failConfig)
+
+	// get current path and create tmp dir
+	fakeBoot := "fakeboot"
+	tempPath, err := builder.TempFile(fakeBoot)
+	if err != nil {
+		return err
+	}
+	pathName := tempPath.Name()
+	fmt.Printf("Temp path: %v\n", pathName)
+	lastElement := path.Base(pathName)
+
+	createBoot, err := os.Create(lastElement)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("createBoot: %v\n", createBoot.Name())
```

Comment:
The file returned by `TempFile` is already an `os.File`, so we don't have to `os.Create` it again or get its filename.